### PR TITLE
Feat/header spacing

### PR DESCRIPTION
**Pull request by TurnbulA**
Sorted the spacing in the header 
Couldn't work out how to make the header text thinner

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,275 +7,221 @@
...
+                    <li>Item 3</li>
+                </ul>
+            </div>
+            <div class="c-nav__section">
+                <h1><em>iPaidThat</em></h1>
+            </div>
+            <div class="c-nav__section">
+                <ul class="ul">
+                    <li>Item 1</li>
+                    <li><button class="button button--round">Item 2</button></li>
```
Comment (mali891) [inline index.html]: Is this `button--round` the same as...

File: index.html [hunk 2]
```diff
@@ -7,275 +7,221 @@
...
-                                </div>
-                                <div class="header-button">
-                                    <button class="button-round">Essayez gratuitement</button>
-                                </div>
-                            </div>
-                        </div>
-                    </div>
+                </div>
+                <div class="header-button">
+                    <button class="button-round">Essayez gratuitement</button>
```
Comment (mali891) [inline index.html:65]: this `button-round`?

**Review by mali891 — APPROVED** [review]
Let's get this one in, and I'll create you a new ticket to work on now. Thanks

File: index.html (hunk 1 above)
Comment (TurnbulA) [reply index.html]: No this is the white button in the nav bar, I'll rename it for clarity

**Pull request merged**